cmd/snap-update-ns/change.go

```diff
@@ -59,26 +60,141 @@ var changePerform func(*Change) ([]*Change, error)
 
 // changePerformImpl is the real implementation of Change.Perform
 func changePerformImpl(c *Change) ([]*Change, error) {
+	var changes []*Change
 	if c.Action == Mount {
+		// We may be asked to bind mount a file, bind mount a directory, mount
+		// a filesystem over a directory or create a symlink (which is abusing
+		// the "mount" concept slightly). That actual operation is performed in
+		// c.lowLevelPerform. Here we just set the stage to make that possible.
+		kind, _ := c.Entry.OptStr("x-snapd.kind")
```

Comment:
Why can we ignore the `ok` return here?

Reply:
Because the default is the empty string which corresponds to a directory entry which doesn't have a `x-snapd.kind` mount option.